understand_data.R

```diff
@@ -0,0 +1,19 @@
+# Glimpse of data and missing value check
```

Comment:
Consider using [Roxygen](http://r-pkgs.had.co.nz/man.html#man-workflow) for documentation for functions.

Then users can type `?understand_data` in their notebooks and get a really pretty view of the documentation for the function, just like functions in other R packages. (try `?na.omit`).

Reply:
Correction: `?some_R_function` works for functions that are part of an R package, so it won't work in this case and that's okay, but it's still great to add the documentation for the function in Roxygen format.